arcgis-ios-sdk-samples/Search/Find place/README.md

```diff
@@ -1,19 +1,51 @@
 # Find place
 
-This sample demonstrates how to use geocode functionality to search for points of interest, around a location or within an extent
+Find places of interest near a location or within a specific area.
 
+![Search function](find-place-1.png)
+![Search results](find-place-2.png)
 
-## How to use the sample
-
-Specify the point of interest in the `Point of Interest` textfield (e.g. Starbucks). For the proximity field you can choose between your current location, any other location or you can leave it blank for a generic search. As soon as you start typing into either of the textfields, the app gives you suggestions. When you hit the search button, the resulting locations are shown on the map. You can tap on a pin to get details about that location. You can also pan around and redo search in the new extent
+## Use case
 
+When getting directions or looking for nearby places, users may only know what the place has ("food"), the type of place ("gym"), or the generic place name ("Starbucks"), rather than the specific address. You can get suggestions and locations for these places of interest (POIs) using a natural language query. Additionally, you can filter the results to a specific area.
 
-![](image1.png)
-![](image2.png)
+## How to use the sample
 
+Choose a type of place in the first field and an area to search within in the second field. Tap the Search button to show the results of the query on the map. Tap on a result pin to show its name and address. If you pan away from the result area, a "Redo search in this area" button will appear. tap it to query again for the currently viewed area on the map.
```

Comment:
```suggestion
Choose a type of place in the first field and an area to search within in the second field. Tap the Search button to show the results of the query on the map. Tap on a result pin to show its name and address. If you pan away from the result area, a "Redo search in this area" button will appear. Tap it to query again for the currently viewed area on the map.
```

Reply:
Thanks for reviewing! I'll make those changes 👍🏼